client/components/button/style.scss

```diff
@@ -102,7 +102,7 @@ button {
 }
 
 // Primary buttons
-.button.is-primary {
+@mixin button-is-primary {
```

Comment:
In this redesign, all the changes between mobile and desktop are made using CSS media queries, so the UI can change accordingly even if the user resizes the window (very uncommon, but probably will be more common with the side-by-side app mode in tablets). This particular mixin is not pretty, but it would be worse changing only this one thing on JS code.